### PR TITLE
Ascanrules: external redirect dots deny listed

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: addOns/ascanrules/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 - Server Side Code Injection scan rule, prevent use of zero when injecting ASP multiplication to avoid false positives (Issue 7107).
+- External Redirect scan rule to detect redirects with dots deny listed
```
Comment (kingthorin) [inline addOns/ascanrules/CHANGELOG.md:14]:
```suggestion
- External Redirect scan rule to detect redirects with dots deny listed.
```

Period for consistency.

**Review by kingthorin — COMMENTED** [review]
The handler is the same in both tests and could be extracted, but not a blocker :wink:

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by ricekot — APPROVED** [review]
Thanks!

**Pull request merged**